src/uu/du/src/du.rs

```diff
@@ -505,26 +507,33 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         summarize,
     )?;
 
+    let files = match matches.get_one::<String>(options::FILE) {
+        Some(_) => matches
+            .get_many::<String>(options::FILE)
+            .unwrap()
+            .map(|s| s.as_str())
+            .collect(),
+        None => vec!["."],
+    };
+
     let options = Options {
         all: matches.get_flag(options::ALL),
         max_depth,
         total: matches.get_flag(options::TOTAL),
         separate_dirs: matches.get_flag(options::SEPARATE_DIRS),
         one_file_system: matches.get_flag(options::ONE_FILE_SYSTEM),
         dereference: matches.get_flag(options::DEREFERENCE),
+        dereference_args: if matches.get_flag(options::DEREFERENCE_ARGS) {
```

Comment:
Maybe these should be one enum?
```rust
enum Deref {
    All,
    Args(Vec<PathBuf>),
    None,
}
```
Or maybe the args should be in the settings and then it would just be
```rust
enum Deref {
    All,
    Args,
    None,
}
```

Reply:
good idea, i will have a look